controller/config.go

```diff
@@ -298,7 +298,11 @@ func LoadConfig(path string) (*Config, error) {
 									return nil, fmt.Errorf("error loading advertiseAddress for [ctrl/options] (%v)", err)
 								}
 								if controllerConfig.Raft != nil {
-									controllerConfig.Raft.AdvertiseAddress = advertiseAddr
+									addr, err := transport.ParseAddress(advertiseAddr)
+									if err != nil {
+										return nil, errors.Wrap(err, "unable to parse [ctrl/options/advertiseAddress]")
```

Comment:
can youprint what the value was that couldn't parse too? seems like a friendly thing to do imo

Reply:
Fixed